### PR TITLE
Add dockerfile for easier development

### PR DESCRIPTION
**Pull request by joonatoona**

### CODE REVIEW

**Comment by RobLoach** [comment]
Would you add a note in the README.md or something about how to use it?!??!!

**Comment by joonatoona** [comment]
Its just a docker container. `docker build -t lutrisweb Docker/` then `docker run -it -p 8000:8000 lutrisweb /bin/bash`

File: README.rst [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
+
+::
+
+    docker run --name lutrisdev -it -p 8000:8000 lutrisweb /bin/bash
+
+Reconnecting after creating
+
+::
+
+    docker start lutisdev
```
Comment (strycore) [inline README.rst]: typo here :D

File: Docker/Dockerfile [hunk 2]
```diff
@@ -10,4 +10,4 @@ RUN apt-get install -y git curl python python-pip imagemagick memcached libmemca
 RUN /bin/bash /install.sh
 RUN rm lutrisweb/settings/base.pyc
 
-CMD ["python", "manage.py runserver"]
+CMD ["/bin/bash"]
```
Comment (strycore) [inline Docker/Dockerfile]: I guess that's temporary until the Dockerfile is finished?

Comment (joonatoona) [inline Docker/Dockerfile]: Yeah, for whatever reason starting it with the dockerfile causes it to crash instantly, but running it though bash on the conatiner works perfectly.

File: README.rst (hunk 1 above)
Comment (joonatoona) [inline README.rst]: Fixed, oops :D

**Comment by strycore** [comment]
Looks good enough to start experimenting with it, going to merge the PR.

**Pull request merged**